### ISSUE TITLE
bug: Keyboard packages have an issue displaying the help file

### ISSUE THREAD

**Issue by LornaSIL** [keymanapp/help.keyman.com#1560]
Keyman package help files have an error message in the help file that it cannot find the **package** `.js` file. Since a package is made up of two or more keyboards it will never find a package `.js`. For example the following package would never have a `gff_gurage_and_amharic.js` file because it contains `gff_gurage.js` and `gff_amharic.js`:

https://help.keyman.com/keyboard/gff_gurage_and_amharic/1.1/gff_gurage_and_amharic
https://help.keyman.com/keyboard/galaxie_greek_hebrew_positional/2.2.1/galaxie_greek_hebrew_positional

![image](https://github.com/user-attachments/assets/24363f46-3d4f-48c9-9484-52640a0be223)

![image](https://github.com/user-attachments/assets/513f4343-399e-43dc-bc1b-d8d802950151)

I do not believe this is a new behavior, I believe I've seen it before, but I never figured out that it was for a package and not a keyboard.

**Comment by mcdurdin** [comment]
Yeah, this has been an issue for a while. Not sure how best to resolve at this point, as the help site doesn't have sufficient context to resolve, but disabling KMW is going to lead to missing renders of keyboards. So ... non-trivial!

**Comment by darcywong00** [comment]
I was looking at a Sentry crash report from help.keyman.com involving a multi-keyboard package


https://help.keyman.com/keyboard/vm_tamil/1.4/vm_tamil

couldn't find the file
https://s.keyman.com/keyboard/vm_tamil/1.4/vm_tamil-1.4.js


I noticed a portion from https://api.keyman.com/keyboard/vm_tamil is
```json
"jsFilename": "vm_tamil_typewriter.js",
    "jsFileSize": 11884,
    "packageIncludes": [
        "documentation",
        "visualKeyboard"
    ],
```

Do we expect jsFilename not to be present for multi-keyboard packages?
I see a .js file also gets synced to

https://downloads.keyman.com/keyboards/gff_gurage_and_amharic/1.1/

**Comment by mcdurdin** [comment]
Per @darcywong00 report, https://github.com/keymanapp/keyman/issues/11645 is probably relevant
